### DISCUSSION TITLE
Context Menus in Tool Windows

### DISCUSSION THREAD

**Discussion by sharpninja** [VsixCommunity/Community.VisualStudio.Toolkit#486] [General]
I need to add a context menu to my tool window, but when I added the `ContextMenu` control it doesn't get styled at all.  What is the correct way to define and add a context menu to a Tool Window?

**Comment by reduckted** [comment]
The correct way to is define the context menu in the `.vsct` file.

There's a tutorial that doesn't use this toolkit here:
https://learn.microsoft.com/en-us/visualstudio/extensibility/adding-a-shortcut-menu-in-a-tool-window?view=vs-2022#create-the-tool-window-shortcut-menu-package

You'd do it much the same way if you use this toolkit, but I can put together an example for you if you need one.
